scripts/MantidIPython/plot_functions.py

```diff
@@ -0,0 +1,111 @@
+"""
+Plotting functions for use in IPython notebooks that are generated by MantidPlot
+"""
+
+import matplotlib.pyplot as plt
+import sys
+import os
+
+# Import Mantid
+CURRENT_DIR = os.getcwd()
```

Comment:
At the moment it is implemented such that you don't need to use the launch script. This is because there is no launch script on Mac yet. I can either, leave this in for now and make a note in my Mac launch script issue (#13728) to clean up this at the same time, or, remove this now on the basis that no one is actually using notebooks on Mac yet anyway.